I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 51, in script
    call screen ticTacToeGame
  File "renpy/common/000statements.rpy", line 671, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
Exception: Screen ticTacToeGame is not known.


-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 51, in script
    call screen ticTacToeGame
  File "C:\Users\jayly\Downloads\renpy-8.3.7-sdk\renpy\ast.py", line 1987, in execute
    self.call("execute")
  File "C:\Users\jayly\Downloads\renpy-8.3.7-sdk\renpy\ast.py", line 1966, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\jayly\Downloads\renpy-8.3.7-sdk\renpy\statements.py", line 374, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 671, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "C:\Users\jayly\Downloads\renpy-8.3.7-sdk\renpy\exports\statementexports.py", line 331, in call_screen
    renpy.exports.show_screen(_screen_name, *args, _transient=True, **kwargs)
  File "C:\Users\jayly\Downloads\renpy-8.3.7-sdk\renpy\display\screen.py", line 1328, in show_screen
    raise Exception("Screen %s is not known.\n" % (name[0],))
Exception: Screen ticTacToeGame is not known.


Windows-10-10.0.26100 AMD64
Ren'Py 8.3.7.25031702
datingsim 1.0
Tue May 20 17:25:12 2025
